spec/models/community_users.rb

```diff
@@ -0,0 +1,84 @@
+require 'spec_helper'
+
+describe CommunityUsers do
+  let(:student) { create(:student) }
+  let(:student_looking_for_coach) { create(:student, :interested_in_coaching) }
+  let(:coach) { create(:coach, :available) }
+  describe '#all' do
+    before do
+      @params = {
```

Comment:
I don't really see the point of switching from `lets` to instance vars here 🙈 

Can you please convert them to `let()` statements?

This also does not need to be in a before block from what I see.